recipes/tomlplusplus/all/test_package/CMakeLists.txt

```
@@ -7,3 +7,6 @@ conan_basic_setup()
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
+if(DEFINED TOMLPP_BUILD_SINGLE_ONLY AND TOMLPP_BUILD_SINGLE_ONLY)
```

```suggestion
if(${TOMLPP_BUILD_SINGLE_ONLY})
```

Could we simplify this?

---

That's not semantically the same but I've never tested the edge cases. 

Consumers should not be passing values so presumptively it's safe

I'll apply if there's any other changes